crypto/evp/evp_pbe.c

```diff
@@ -173,7 +173,10 @@ int EVP_PBE_alg_add_type(int pbe_type, int pbe_nid, int cipher_nid,
     pbe_tmp->md_nid = md_nid;
     pbe_tmp->keygen = keygen;
 
-    sk_EVP_PBE_CTL_push(pbe_algs, pbe_tmp);
+    if (!sk_EVP_PBE_CTL_push(pbe_algs, pbe_tmp)) {
+        OPENSSL_free(pbe_tmp);
```

Comment:
I was thinking that the sk_TYPE_push() was already pushing an `Err` .
I was wrong,
I will do a separate PR about adding the missing `Err` later.